pkg/installer/storageclass.go

```diff
@@ -20,134 +20,154 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"io"
 
 	directpvtypes "github.com/minio/directpv/pkg/apis/directpv.min.io/types"
+	"github.com/minio/directpv/pkg/consts"
 	"github.com/minio/directpv/pkg/k8s"
 	corev1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
 	storagev1beta1 "k8s.io/api/storage/v1beta1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
 var errStorageClassVersionUnsupported = errors.New("unsupported StorageClass version found")
 
-func installStorageClassDefault(ctx context.Context, c *Config) error {
-	if err := executeFn(ctx, c, "storage class", createStorageClassDefault); err != nil {
-		return fmt.Errorf("unable to create storage class; %v", err)
-	}
-	return nil
-}
-
-func uninstallStorageClassDefault(ctx context.Context, c *Config) error {
-	if err := executeFn(ctx, c, "storage class", deleteStorageClassDefault); err != nil {
-		return fmt.Errorf("unable to delete storage class; %v", err)
+func createStorageClass(ctx context.Context, args *Args) error {
+	var gvk *schema.GroupVersionKind
+	if args.DryRun {
+		if args.KubeVersion.Major() >= 1 && args.KubeVersion.Minor() < 16 {
```

Comment:
```suggestion
		if args.KubeVersion.Major() == 1 && args.KubeVersion.Minor() < 16 {
```

Reply:
No. Any non-standard kubernetes, we need to use lower version for safer side.